apps/web/lib/api/projects.ts

```diff
@@ -128,11 +126,9 @@ export async function deleteProjectAdmin(
   ]);
 
   const deleteProjectResponse = await Promise.all([
-    // delete project logo from Cloudinary
-    project.logo &&
-      cloudinary.v2.uploader.destroy(`logos/${project.id}`, {
-        invalidate: true,
-      }),
+    // delete project logo if it's a custom logo stored in R2
+    project.logo?.startsWith(`https://${process.env.STORAGE_DOMAIN}`) &&
```

Comment:
I think we should be consistent with `STORAGE_DOMAIN` and `STORAGE_ENDPOINT` in terms of whether or not it has the `https` protocol on it. `STORAGE_ENDPOINT` will always have it so we should make `STORAGE_DOMAIN` also have it. Since domain isn't the right now I can rename to `STORAGE_BASE_URL`

Reply:
agreed, I think having `STORAGE_BASE_URL` makes much more sense here :thumbs-up: